extended/src/main/java/apoc/config/Config.java

```diff
@@ -14,16 +14,73 @@
 import org.neo4j.procedure.Procedure;
 
 import java.util.Map;
+import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static apoc.ApocConfig.APOC_CONFIG_JOBS_POOL_NUM_THREADS;
+import static apoc.ApocConfig.APOC_CONFIG_JOBS_QUEUE_SIZE;
+import static apoc.ApocConfig.APOC_CONFIG_JOBS_SCHEDULED_NUM_THREADS;
+import static apoc.ApocConfig.APOC_EXPORT_FILE_ENABLED;
+import static apoc.ApocConfig.APOC_IMPORT_FILE_ALLOW__READ__FROM__FILESYSTEM;
+import static apoc.ApocConfig.APOC_IMPORT_FILE_ENABLED;
+import static apoc.ApocConfig.APOC_IMPORT_FILE_USE_NEO4J_CONFIG;
+import static apoc.ApocConfig.APOC_TRIGGER_ENABLED;
+import static apoc.ExtendedApocConfig.APOC_TTL_ENABLED;
+import static apoc.ExtendedApocConfig.APOC_TTL_LIMIT;
+import static apoc.ExtendedApocConfig.APOC_TTL_SCHEDULE;
+import static apoc.ExtendedApocConfig.APOC_UUID_ENABLED;
+import static apoc.ExtendedApocConfig.APOC_UUID_FORMAT;
+import static apoc.custom.CypherProceduresHandler.CUSTOM_PROCEDURES_REFRESH;
+import static apoc.trigger.TriggerHandler.TRIGGER_REFRESH;
+import static apoc.util.Util.HTTP_TIMEOUT_CONNECT_CONFIG;
+import static apoc.util.Util.HTTP_TIMEOUT_READ_CONF;
+
 /**
  * @author mh
  * @since 28.10.16
  */
 @Extended
 public class Config {
+    private static final Set<String> WHITELIST_CONFIGS = Set.of(
+            // apoc.import.
+            APOC_IMPORT_FILE_ENABLED,
+            APOC_IMPORT_FILE_USE_NEO4J_CONFIG,
+            APOC_IMPORT_FILE_ALLOW__READ__FROM__FILESYSTEM,
+
+            // apoc.export.
+            APOC_EXPORT_FILE_ENABLED,
+            
+            // apoc.trigger.
+            APOC_TRIGGER_ENABLED,
+            TRIGGER_REFRESH,
+
+            // apoc.uuid.
+            APOC_UUID_ENABLED,
+            APOC_UUID_FORMAT,
+            
+            // apoc.ttl.
+            APOC_TTL_SCHEDULE,
+            APOC_TTL_ENABLED,
+            APOC_TTL_LIMIT,
+            
+            // apoc.jobs.
+            APOC_CONFIG_JOBS_SCHEDULED_NUM_THREADS,
+            APOC_CONFIG_JOBS_POOL_NUM_THREADS,
+            APOC_CONFIG_JOBS_QUEUE_SIZE,
+            
+            // apoc.http.
+            HTTP_TIMEOUT_CONNECT_CONFIG,
+            HTTP_TIMEOUT_READ_CONF,
+            
+            // apoc.custom.
+            CUSTOM_PROCEDURES_REFRESH,
 
+            // apoc.spatial. - other configs can have sensitive credentials
+            "apoc.spatial.geocode.osm.throttle",
+            "apoc.spatial.geocode.google.throttle"
```

Comment:
Scusa perché queste due non stanno come statiche?

Reply:
Perché vengono ricavate tramite una concatenazione di stringhe partendo da questa:
```
public static final String PREFIX = "apoc.spatial.geocode";`
```
e poi fa quest: https://github.com/neo4j/apoc/blob/dev/core/src/main/java/apoc/spatial/Geocode.java#L292.

Comunque quel PREFIX di Geocode non è visibile in `extended`, perché si trova in `core`, mentre le altre stanno in `common`.

Reply:
Va beh almeno metti il prefix ok?

Reply:
Come concatenazione intendo